content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -180,63 +281,108 @@ program to retrieve the contents of your Secret.
    [decoding a Secret](/docs/tasks/configmap-secret/managing-secret-using-kubectl/#decoding-secret)
    to completely decode the Secret.
 
-## Ensure all Secrets are encrypted
+### Ensure all Secrets are encrypted
+
+You have configured your cluster so that Secrets are encrypted on write. Performing
+an update on Secret will encrypt that content at rest.
 
-Since Secrets are encrypted on write, performing an update on a Secret will encrypt that content.
+You can make this change across all Secrets in your cluster:
 
 ```shell
+# Run this as an administrator that can read and write all Secrets
 kubectl get secrets --all-namespaces -o json | kubectl replace -f -
 ```
 
-The command above reads all Secrets and then updates them to apply server side encryption.
+The command above reads all Secrets and then updates them with the same data, in order to
+apply server side encryption.
 
 {{< note >}}
 If an error occurs due to a conflicting write, retry the command.
-For larger clusters, you may wish to subdivide the secrets by namespace or script an update.
+It is safe to run that command more than once.
+
+For larger clusters, you may wish to subdivide the Secrets by namespace,
+or script an update.
 {{< /note >}}
 
+## Cleanup {#cleanup-all-secrets-encrypted}
+
+You **must** ensure that all existing Secrets are encrypted at rest before
+performing this cleanup step, because this change prevents the API server from
+retrieving Secret resources (API path `secrets`) that are not encrypted.
+
+Once all Secrets in your cluster are encrypted, you can remove the `identity`
+part of the encryption configuration. For example:
+
+```yaml
+---
+apiVersion: apiserver.config.k8s.io/v1
+kind: EncryptionConfiguration
+resources:
+  - resources:
+      - secrets
+    providers:
+      - aescbc:
+          keys:
+            - name: key1
+              secret: <BASE 64 ENCODED SECRET>
+      - identity: {} # REMOVE THIS LINE
+```
+
+…and then restart each API server in turn. This change prevents the API server
+from accessing a plain-text Secret, even by accident.
```

Comment:
This may and may not be a good practice. It depends.
If I'm using one provider today and I'm migrating to a different provider tomorrow, keeping `identity` there may be convenient. I don't know. My gut feeling is that an assertion like this should be made carefully.

Reply:
When is it not a good practice to remove the `identity` provider?